assets/app/styles/_components.less

```diff
@@ -58,6 +58,13 @@
       .component-label-details {
         text-transform: none;
       }
+      a:not(.keep-link-style) {
+        color: @gray-light;
```

Comment:
I'm wondering if we can add something subtle to the style to indicate it's clickable. Maybe a gray dotted underline? Although we use that to mean hover in other places.